src/server/api/routes/products.router.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const productsController = require('../controllers/products.controller');
+
+/**
+ * @swagger
+ * /products:
+ *  get:
+ *    tags:
+ *    - Products
+ *    summary: Get all products
+ *    description:
+ *      Will return all products.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  productsController
+    .getProducts()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /products/{ID}:
+ *  get:
+ *    tags:
+ *    - Products
+ *    summary: Get product by ID
+ *    description:
+ *      Will return single product with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the module to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
```

Comment:
based on productsController, this doesn't only return 200 or 500, but also 400 and 404 if the id isn't correct.

I would say we should include those options in the responses section of swagger